### DISCUSSION TITLE
does not work on pages with custom domain URLs

### DISCUSSION THREAD

**Discussion by dgblackout** [AprilSylph/XKit-Rewritten#671] [Help Desk]
When on a tumblr users page with their own domain, nothing loads.

**Comment by AprilSylph** [comment]
XKit Rewritten doesn't do anything to _anyone's_ blog, it only modifies the dashboard.

**Reply by AprilSylph** [reply]
@dgblackout What feature(s) were you expecting to work on the blog network?

**Reply by dgblackout** [reply]
One click queue, reblog, draft would be nice, on post pages.
